### DISCUSSION TITLE
Cascader

### DISCUSSION THREAD

**Discussion by XJY-1** [ant-design/ant-design#36408] [General]
Cascader 级联选择，选择之后清除选择内容，placeholder 消失不见了

**Comment by yoyo837** [comment]
上在线链接

**Comment by yoyo837** [comment]
看看value是不是给null，改为给undefined试试

**Reply by XJY-1** [reply]
清空之后 value 是一个空数组，赋值为 undefined 也不行
我在 CodeSandbox 试了一下是正常的，在项目中会有这个问题，是 antd 的版本为题吗？
我用的版本是 "antd": "4.17.2", 我看更新日志也没有这方面的问题更新

**Reply by yoyo837** [reply]
上在线链接

**Reply by XJY-1** [reply]
https://codesandbox.io/s/cascader-qing-chu-zhi-hou-placeholder-xiao-shi-0fqd50?file=/demo.js

antd 4.17.2

**Reply by afc163** [reply]
https://github.com/ant-design/ant-design/issues/32939#issuecomment-1015220871

https://codesandbox.io/s/cascader-qing-chu-zhi-hou-placeholder-xiao-shi-forked-2f9kw3?file=/package.json

**Reply by XJY-1** [reply]
好的 谢谢 这就去安装最新的
